Add `Donut Cup` wiki article (#11688)
ppy/osu-wiki: wiki/Tournaments/Donut_Cup/1/en.md

![Donut Cup banner](img/banner.jpg)

The **Donut Cup** is double-elimination 2v2 osu! tournament hosted by ::{ flag=CA }:: [KKipalt](https://osu.ppy.sh/users/6889573) and ::{ flag=US }:: [Rentai](https://osu.ppy.sh/users/11033243). The tournament is open to teams of osu! players with a global rank of #1,000 or lower after [badge-weighted seeding (BWS) calculations](/wiki/Tournaments/Badge-weighted_seeding). Although it is the first to be held under the Donut Cup name, the tournament is considered a spiritual successor to the previously held [Bamboo Cup](https://osu.ppy.sh/community/forums/topics/1274066) and [Crepe Cup](https://osu.ppy.sh/community/forums/topics/1538394).

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2024-04-14/2024-05-12 |
| Screening phase | 2024-05-13/2024-05-26 |
| Qualifiers | 2024-05-27/2024-06-02 |
| Round of 32 | 2024-06-03/2024-06-09 |
| Round of 16 | 2024-06-10/2024-06-16 |
| Quarterfinals | 2024-06-17/2024-06-23 |
| Semifinals | 2024-06-24/2024-06-30 |
| Finals[^stage-note-1] | 2024-07-01/2024-07-07 |
| Grand Finals[^stage-note-2] | 2024-07-08/2024-07-15 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 1 month of osu!supporter for each team member, custom-made profile banner, a batch of freshly-baked donuts (conditional according to feasibility of the delivery) |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 1 month of osu!supporter for each team member, custom-made profile banner, a batch of freshly baked donuts (conditional according to feasibility of the delivery) |
```

- Hard Rock
  1. [Reol - Saisaki (cRyo\[iceeicee\]) \[Extra\]](https://osu.ppy.sh/beatmapsets/858075#osu/1793422)
  2. [KanoeRana - 45323 (Airiesu) \[me/you\]](https://osu.ppy.sh/beatmapsets/1347840#osu/2791200)
  3. [AVTechNO! - boku-boku SuketchP Remix (feat. Hatsune Miku) (rollpan) \[Extreme - OTST\]](https://osu.ppy.sh/beatmapsets/1829854#osu/3755660)
- Double Time
  1. [Ceui - Hoshi Ori Yume Mirai (Hishiro Chizuru) \[Cyndere's Expert\]](https://osu.ppy.sh/beatmapsets/1517513#osu/3136056)
  2. [senya - Kanjou Chemistry (Drum 'n' Bass Remix) (Skubi) \[PAJWOJ's Vestige of Days Gone By\]](https://osu.ppy.sh/beatmapsets/2084917#osu/4384262)
  3. [fripSide - only my railgun -Euro Beat Charger Mix- (Nos\_If) \[Insane\]](https://osu.ppy.sh/beatmapsets/1558910#osu/3184307)
  4. [Susumu Hirasawa - SWITCHED-ON LOTUS (Starrodkirby86) \[KIRBY Mix Deluxe\]](https://osu.ppy.sh/beatmapsets/16457#osu/58970)
- Free Mod
  1. [Tokyo Jihen - Jusui Negai -The Suicide- (seros) \[Kyouiku\]](https://osu.ppy.sh/beatmapsets/1761649#osu/3605664)
  2. [Tsukifune Kosho-do - Tabi no Owari to Shukusai no Yoru (vergil chair) \[Journey\]](https://osu.ppy.sh/beatmapsets/1346576#osu/2788659)

>   3. [paraoka - rampage (kiri09) \[9\]](https://osu.ppy.sh/beatmapsets/113359#osu/294086)  

Reviewer: ```suggestion
  3. [paraoka - rampage (kiri09) \[9\]](https://osu.ppy.sh/beatmapsets/113359#osu/294086)
```

  2. [SHIKI - Pure Ruby (Pennek) \[new ar8\]](https://osu.ppy.sh/beatmapsets/1347084#osu/4650726)
  3. [MK - Irresistible (Cubby) \[Expert\]](https://osu.ppy.sh/beatmapsets/1900852#osu/3918333)
- Hard Rock
  1. [THE ORAL CIGARETTES - Flower (pti) \[Wither\]](https://osu.ppy.sh/beatmapsets/2020993#osu/4208843)
  2. [Shpongle - Levitation Nation (piroshki) \[Welcome to... Brazil?!\]](https://osu.ppy.sh/beatmapsets/1801375#osu/3693390)
  3. [HyuN - Illusion of Inflict (Icekalt) \[N/A's Extra\]](https://osu.ppy.sh/beatmapsets/968923#osu/2109396)
- Double Time
  1. [WEAVER - Kuchizuke Diamond (Net0) \[Happy Birthday\]](https://osu.ppy.sh/beatmapsets/1103117#osu/2304350)
  2. [MORE MORE JUMP! x Hatsune Miku - Idol Shin'etai (xidorn) \[BURST\]](https://osu.ppy.sh/beatmapsets/2195379#osu/4645182)
  3. [fripSide - flower of bravery (ImpurePug) \[Insane\]](https://osu.ppy.sh/beatmapsets/1824641#osu/3744397)
  4. [RuLu feat. Youi - Xylophone (Irin) \[Fish Rose's Insane\]](https://osu.ppy.sh/beatmapsets/1352063#osu/2892338)
- Free Mod
  1. [the liquid movement - Love sick (mintIceCream\_) \[tv\]](https://osu.ppy.sh/beatmapsets/2138395#osu/4654833)

>   2. [Kikuo - Yousei no Kiri (Vararaup) \[Damnjelly's Magical\]](https://osu.ppy.sh/beatmapsets/1880072#osu/3870421)  

Reviewer: ```suggestion
  2. [Kikuo - Yousei no Kiri (Vararaup) \[Damnjelly's Magical\]](https://osu.ppy.sh/beatmapsets/1880072#osu/3870421)
```

- Hidden
  1. [My Chemical Romance - Cemetery Drive (aishiteiru-) \[Mirash's Fall\]](https://osu.ppy.sh/beatmapsets/1081808#osu/2842445)
  2. [Cascada - Bad Boy (Nightcore Mix) (winber1) \[Extreme\]](https://osu.ppy.sh/beatmapsets/41095#osu/130030)
  3. [II-L - V804N (BMS Edit) (Not Airwam) \[isca101's 3XTR4\]](https://osu.ppy.sh/beatmapsets/1918888#osu/4065173)
- Hard Rock
  1. [AAAA - Hoshizora no Kanransha (Setu) \[Conclusion\]](https://osu.ppy.sh/beatmapsets/2094949#osu/4393494)
  2. [AAAA - carnation (Mirash) \[Mirashola Fanboy's Expert\]](https://osu.ppy.sh/beatmapsets/1893490#osu/3934268)
  3. [midwxst - Smile (feat. glaive) (- ascended -) \[Feign\]](https://osu.ppy.sh/beatmapsets/1458846#osu/2997929)
- Double Time
  1. [Kurosaki Maon - Rakuen no Tsubasa (Asagi) \[Insane\]](https://osu.ppy.sh/beatmapsets/227041#osu/529108)
  2. [Hanazono Mei - Kurenai Fubuki (Nao Tomori) \[Collab Insane\]](https://osu.ppy.sh/beatmapsets/1549881#osu/3167236)
  3. [lapix - Flamenco House (Luminiscental) \[Insane\]](https://osu.ppy.sh/beatmapsets/1865291#osu/3857255)
- Free Mod
  1. [PUP - Factories (Sparxo) \[THROUGH THE NIGHT\]](https://osu.ppy.sh/beatmapsets/1239345#osu/2576707)

>   2. [inabakumori - Tokoshizume  (xyperX) \[Kard's Insane\]](https://osu.ppy.sh/beatmapsets/1777533#osu/3657734)

Reviewer: ```suggestion
  2. [inabakumori - Tokoshizume (xyperX) \[Kard's Insane\]](https://osu.ppy.sh/beatmapsets/1777533#osu/3657734)
```

### Qualifiers

- No Mod
  1. [Harano Oni - Yomei Senkoku (Dailycare) \[Normal\]](https://osu.ppy.sh/beatmapsets/2031675#osu/4234684)
  2. [F-777 - Vortex (knowledgeking) \[THREE DIMENSIONS\]](https://osu.ppy.sh/beatmapsets/1543390#osu/3154777)
  3. [shoujo byou - climactic cry (Crystal) \[dedicated chivalry\]](https://osu.ppy.sh/beatmapsets/1959554#osu/4060450)
  4. [AJURIKA - BIG som tam (KKipalt) \[:drool:\]](https://osu.ppy.sh/beatmapsets/1833709#osu/3764497)
- Hidden
  1. [yak\_won - Wind Tales (dakiwii) \[catri\]](https://osu.ppy.sh/beatmapsets/1952375#osu/4042950)
  2. [I love you orchestra - Coconut Love (funny) \[Nucifera\]](https://osu.ppy.sh/beatmapsets/1665501#osu/3400091)
- Hard Rock
  1. [Oohashi Takuya (from Sukima Switch) - Katamari on the Wings (melon boy) \[OK MR. SUNSHINE\]](https://osu.ppy.sh/beatmapsets/1867650#osu/3841928)
  2. [Rabbit House - das Leben nach dem Tode (Rentia) \[Unterwelt\]](https://osu.ppy.sh/beatmapsets/1733348#osu/3542916)
- Double Time
  1. [ICHIKO - I SAY YES (N a s y a) \[Insane\]](https://osu.ppy.sh/beatmapsets/2179900#osu/4604218)

>   2. [yanaginagi - Sentimental (defiance) \[Insane\]](https://osu.ppy.sh/beatmapsets/1562788#osu/3191336)  

Reviewer: ```suggestion
  2. [yanaginagi - Sentimental (defiance) \[Insane\]](https://osu.ppy.sh/beatmapsets/1562788#osu/3191336)
```

4. Teams who can compete in the next round are determined by:
   - In the Round of 32 and the Round of 16, each team needs to win 5 points in order to win a match. (Best of 9)
   - In the Quarterfinals and the Semifinals, each team needs to win 6 points in order to win a match. (Best of 11)
   - In the Finals[^stage-note-1] onwards, each team needs to win 7 points in order to win a match. (Best of 13)
   - Whether there are teams who are declared to win the match by default.
   - Whether there are teams who are disqualified from the tournament.

### Match regulations

1. Prior to starting the match, representatives from each team must run the `!roll` command in order to determine the banning and picking order.
   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
   - The loser of the `!roll` gets the opposite by default.
   - This rule does not apply in the Qualifier lobbies.
2. Each team has to ban **two beatmaps** from the corresponding mappool. These beatmaps will not be allowed to be picked by any team during the entire match.
   - The bans will be done alternatingly (A–B–A–B) between the two teams.

>    - Barring the tiebreaker, there are no restrictions as to which maps may and may not be banned in a match.

Reviewer: more natural

```suggestion
   - Barring the tiebreaker, there are no restrictions as to which maps may or may not be banned in a match.
```

3. Each team will be given a chance to pick **one warm-up beatmap** (up to 4:30 max) to be played in the lobby.
   - Playing a warm-up beatmap is not mandatory, and teams may elect to skip their warm-up pick should they wish to.
4. Teams are expected to exercise common sense in pick time windows.

>    - If a team is unable to come up with a pick within a 90–second time window of their picking turn, the pick will be given to the other team.    

Reviewer: ```suggestion
   - If a team is unable to come up with a pick within a 90-second time window of their picking turn, the pick will be given to the other team.
```

5. In a Free Mod pick, each team has to field **at least two different mod combinations** to play the beatmap with.

>    - No Mod doesn't count as an unique mod combination.

Reviewer: ```suggestion
   - No Mod does not count as an unique mod combination.
```

   - Hard Rock and Hidden + Hard Rock counts as the same mod combination.
   - There are no bonus score multipliers for players who are playing with either Easy or Flashlight.
6. In the case of a tiebreaker, the tiebreaker map will be played with the Free Mod option enabled.
   - Playing the tiebreaker map with a mod is *not* mandatory.
7. The results of each match and any other relevant information regarding the match will be noted by the referee after the match has been concluded.

> 

Reviewer: ```suggestion

## Notes

```
